README_GITHUB_ACTIONS.md

```diff
@@ -0,0 +1,38 @@
+# GitHub Actions Notes for Maintainers
+
+rocket-chip makes use of [GitHub Actions](https://github.com/features/actions)
+for running continuous integration (CI) tests. For more information on GitHub
+Actions, please see their [documentation](https://help.github.com/en/actions).
+
+
+## How It Is Configured
+
+The main configuration file describing the continuous integration workflow is
+located at
+[.github/workflows/continuous-integration.yml](.github/workflows/continuous-integration.yml).
```

Comment:
If this YML file is only for GitHub Actions, is it possible to add as a comment to the top of that YML file itself? To avoid confusion for someone trying to tweak the travis job?

Reply:
Yes, this is only for GitHub Actions (which requires the file to be located at `.github/workflows`). Added a comment in 3becccb04f30d679d347f3c27305510660bb7e5d.